examples/resources/base.py

```diff
@@ -14,12 +14,12 @@ def health_auth_check() -> Dict:
 
 
 @app.get('/raise_cuenca_errors')
-def raise_cuenca_errors() -> NoReturn:
+def raise_cuenca_errors() -> None:
```

Comment:
según mypy  el tipo correcto de retorno es `NoReturn`
https://mypy.readthedocs.io/en/latest/more_types.html#the-noreturn-type